flambe/vision/classification/model.py

```diff
@@ -0,0 +1,55 @@
+import torch
+from torch import Tensor
+
+from typing import Optional, Tuple, Union
+from flambe.nn import Module
+
+
+class ImageClassifier(Module):
+    """Implements a simple image classifier.
+
+    This classifier consists of an encocder module, followed by
+    a fully connected output layer that outputs a probability
+    distribution.
+
+    Attributes
+    ----------
+    encoder: Moodule
+        The encoder layer
+    linear: Module
+        The fully connected linear layer
+    output_layer: Module
+        The output layer, yields a probability distribution over targets
+    """
+    def __init__(self,
+                 encoder: Module,
+                 linear: Module,
+                 output_layer: Module) -> None:
+        super().__init__()
+
+        self.encoder = encoder
+        self.linear = linear
```

Comment:
why the extra linear layer if there is already a linear output layer?

Reply:
You could only have one fully connected layer with `SoftmaxLayer`, so i added this to add more layers. I could also modify `SoftmaxLayer` so that i could create multiple fully connected layers. Let me know which you prefer.

Reply:
I guess to me if you wanted more fully connected layers, it seems like it would be better to have them in the encoder, but i understand that makes it weird with the CNNEncoder. I think it might make more sense to have the output layer be the one to add extra linear layers.

Reply:
basically you could do:

```
output_layer: !MLPEncoder
    input_size: ...
    output_size: ...
    n_layers: 2
    output_activation: !torch.LogSoftmax
```
So i think the linear argument can be removed

Reply:
Will fix in the next release. Let's merge as is